### DISCUSSION TITLE
export/import settings

### DISCUSSION THREAD

**Discussion by xitex** [nicumicle/simple-jwt-login#45] [Ideas]
It would be convenient to be able to import export settings in json format.
Thank you!

**Comment by nicumicle** [comment]
I Agree. I'm currently working on this feature. In a couple of days, it will be available.

**Reply by xitex** [reply]
Hi Friend! How are you? Any news?

**Reply by nicumicle** [reply]
Hello,

Please check this repository: https://github.com/simple-jwt-login/export-import

Currently, it is not published in the WordPress.org plugins directory. It will take ~1 week to be there.

To install it, just download it from GitHub as a zip and upload the zip file to your WordPress.

Please let me know if this fits your needs.

Best regards,
Nicu.
